lib/index.js

```diff
@@ -319,5 +319,6 @@ module.exports = {
     requirements: {
         hapi: '>=18.0.0'
     },
-    pkg: require('../package.json')
+    pkg: require('../package.json'),
+    Stdlogger
```

Comment:
I think the standard way to export something in addition to a plugin is to structure it like this:
```js
exports.plugin = { register, requirements, pkg };

exports.Stdlogger = Stdlogger;
```

hapi knows to check for a `plugin` property like this 👍

Reply:
Updated